pylot/debug/imu_visualizer_operator.py

```diff
@@ -0,0 +1,77 @@
+"""Implements an operator that visualizes forward linear acceleration."""
```

Comment:
No need to include this file because we're already visualizing IMU in the VisualizerOperator.